src/commands/slash/suggest.ts

```diff
@@ -0,0 +1,81 @@
+import type { APIEmbed } from "discord-api-types/payloads";
+import { ApplicationCommandType } from "discord-api-types/payloads";
+import { SlashCommand } from "../../utils/SlashCommand";
+import { config } from "../../config";
+import { Logger } from "../../utils/Logger";
+import type { GuildTextBasedChannel } from "discord.js";
+import { MessageActionRow } from "discord.js";
+
+export const command = new SlashCommand(
+	"suggest",
+	async (interaction) => {
+		try {
+			if (!interaction.isCommand() || !interaction.inCachedGuild()) return;
+
+			await interaction.deferReply({
+				ephemeral: true,
+			});
+
+			const channel = interaction.guild.channels.cache.get(
+				config.logs.suggestions
+			) as GuildTextBasedChannel;
+
+			const embed: APIEmbed = {
+				title: "New Suggestion!",
+				description: interaction.options.data[0].value as string,
+				footer: {
+					text: `Suggested by ${interaction.member.user.tag}`,
+					icon_url: interaction.member.displayAvatarURL(),
+				},
+				color: config.commandsEmbedsColor,
+			};
+
+			const row = new MessageActionRow().addComponents([
+				{
+					customId: "suggestion-accept",
+					disabled: false,
+					emoji: "✅",
+					label: "Accept",
+					style: "SUCCESS",
+					type: "BUTTON",
+				},
+				{
+					customId: "suggestion-decline",
+					disabled: false,
+					emoji: "❌",
+					label: "Decline",
+					style: "DANGER",
+					type: "BUTTON",
+				},
+			]);
```

Comment:
Pt. 2 (run prettier after this) (why tf don't you use the MessageButton class)
```suggestion
			const row = new MessageActionRow().addComponents(
				{
					customId: "suggestion-accept",
					disabled: false,
					emoji: "✅",
					label: "Accept",
					style: "SUCCESS",
					type: "BUTTON",
				},
				{
					customId: "suggestion-decline",
					disabled: false,
					emoji: "❌",
					label: "Decline",
					style: "DANGER",
					type: "BUTTON",
				}
			);
```

Reply:
Ah